### PR TITLE
Fixing issues

### PR DESCRIPTION
**Pull request by scottdtaylor95**

### CODE REVIEW

**Comment by aarmey** [comment]
@scottdtaylor95 is this ready to review?

**Comment by scottdtaylor95** [comment]
@aarmey Yes, just added the new figure svg file.

File: tfac/figures/figure3.py [hunk 1]
```diff
@@ -1,23 +1,68 @@
...
 
 def makeFigure():
     """ Get a list of the axis objects and create a figure. """
     # Get list of axis objects
-    ax, f = getSetup((8, 8), (1, 1))
-    # Add subplot labels
-    subplotLabel(ax)
+    ax, f = getSetup((15, 18), (3, 1))
+    pears, ser, plas = fig_3_setup()
+    a = sns.pointplot(data=pears, x=0, y=1, join=False, ax=ax[0])
```
Comment (aarmey) [inline tfac/figures/figure3.py:45]: I wonder if this can be more compact by putting the cytokines on the y-axis?

File: tfac/figures/figure3.py [hunk 2]
```diff
@@ -1,23 +1,68 @@
...
+    return pears, ser, plas
 
 
 def makeFigure():
     """ Get a list of the axis objects and create a figure. """
     # Get list of axis objects
-    ax, f = getSetup((8, 8), (1, 1))
-    # Add subplot labels
-    subplotLabel(ax)
+    ax, f = getSetup((15, 18), (3, 1))
```
Comment (aarmey) [inline tfac/figures/figure3.py:43]: This size should not be larger than a width of 8 in and a height of 6 in.

File: tfac/figures/figure3.py [hunk 3]
```diff
@@ -1,23 +1,68 @@
...
+    pears, ser, plas = fig_3_setup()
+    a = sns.pointplot(data=pears, x=0, y=1, join=False, ax=ax[0])
+    a.set_xticklabels(a.get_xticklabels(), rotation=45, fontsize=12, ha="right")
+    a.tick_params(axis='y', labelsize=12)
+    a.set_ylabel("Pearson's correlation", fontsize = 15)
```
Comment (aarmey) [inline tfac/figures/figure3.py:48]: Do not define font sizes in individual figures. These should be controlled globally by the theme.

File: tfac/figures/figure3.py [hunk 4]
```diff
@@ -1,23 +1,68 @@
...
+    a.set_title("Serum-Plasma Cytokine Level Correlation", fontsize=15)
+    b = sns.boxplot(data=ser, x="variable", y="value", hue="Outcome", ax=ax[1])
+    handles, _ = b.get_legend_handles_labels()
+    b.legend(handles, ["Resolved", "Persisted"], fontsize=12)
+    b.set_xticklabels(b.get_xticklabels(), rotation=45, fontsize=12, ha="right")
+    b.tick_params(axis='y', labelsize=12)
+    b.set_ylabel("Normalized cytokine level", fontsize = 15)
+    b.set_xlabel("Cytokine", fontsize=15)
+    b.set_title("Normalized Serum Cytokine Level by Outcome", fontsize=15)
+    c = sns.boxplot(data=plas, x="variable", y="value", hue="Outcome", ax=ax[2])
```
Comment (aarmey) [inline tfac/figures/figure3.py:59]: I wonder if these can be transposed to having the cytokines on the y-axis, too. Also, perhaps both cytokine amounts plots can be put together? In other words, have colors for serum-resolver, serum-persist, plasma-resolver, and plasma-persist.

**Review by aarmey — CHANGES_REQUESTED** [review]
Some edits.

File: tfac/figures/figure3.py (hunk 1 above)
Comment (scottdtaylor95) [reply tfac/figures/figure3.py:45]: Do you mean essentially rotate the 3 graphs 90 degrees? Or switch the axes and change the dimensions/sizing?

File: tfac/figures/figure3.py (hunk 2 above)
Comment (scottdtaylor95) [reply tfac/figures/figure3.py:43]: To clarify, the largest allowable size is 8x6? Is 6x8 okay?

File: tfac/figures/figure3.py (hunk 3 above)
Comment (scottdtaylor95) [reply tfac/figures/figure3.py:48]: Do you mean globally across all figures, or subfigures? As in should I add a line setting the theme inside figure 3, or is this something you are doing elsewhere?

Comment (aarmey) [reply tfac/figures/figure3.py:48]: Across all figures. You really shouldn't have to change the theme, either. I'd remove all these font size arguments and then see what it looks like.

File: tfac/figures/figure3.py (hunk 1 above)
Comment (aarmey) [reply tfac/figures/figure3.py:45]: seaborn-boxplot-6

File: tfac/figures/figure3.py (hunk 2 above)
Comment (aarmey) [reply tfac/figures/figure3.py:43]: A height of 6 in still allows some body text on a page. The other way around would require a full page figure.

Comment (scottdtaylor95) [reply tfac/figures/figure3.py:43]: Understood, I thought it was scaled down regardless of the size. I'll make sure to adjust that.

File: tfac/figures/figure3.py (hunk 3 above)
Comment (scottdtaylor95) [reply tfac/figures/figure3.py:48]: The issue is probably due to the larger size parameters, I'll see what it looks like in an 8x6 size

**Comment by scottdtaylor95** [comment]
@aarmey I made 4 versions of this - I'm not sure any of them are great given the size we have to work with. Thoughts?
Screenshot (305)
Screenshot (304)
Screenshot (303)
Screenshot (306)

**Comment by aarmey** [comment]
Let's make a full-page supplementary figure, and then we can just pick out select cytokines in the main figure. Just pick 5 for now, and we can make a correct list later.

**Comment by scottdtaylor95** [comment]
Alright, I'll make the main figure and a separate supplementary figure that's 8x8. Which of the above formats would you prefer each to be in?

**Comment by aarmey** [comment]
5 for all. Use your judgement on format.

**Comment by scottdtaylor95** [comment]
I believe this should be ready now.

**Review by aarmey — APPROVED** [review]
Ok great!

**Pull request merged**